Stellar-transaction.x

```diff
@@ -585,19 +585,19 @@ struct InvokeHostFunctionOp
     SorobanAuthorizationEntry auth<>;
 };
 
-/* Bump the expiration ledger of the entries specified in the readOnly footprint
-   so they'll expire at least ledgersToExpire ledgers from lcl.
+/* Extend the TTL of the entries specified in the readOnly footprint
+   so they will live at least extendTo ledgers from lcl.
 
     Threshold: med
-    Result: BumpFootprintExpirationResult
+    Result: ExtendFootprintTTLResult
 */
-struct BumpFootprintExpirationOp
+struct ExtendFootprintTTLOp
 {
     ExtensionPoint ext;
-    uint32 ledgersToExpire;
+    uint32 extendTo;
```

Comment:
Does this mean that the field semantics changes to an absolute value and not relative?

Reply:
i.e. is this just a rename?